x-pack/plugins/canvas/public/lib/sync_filter_expression.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
```

Comment:
nit: add a comment to the right explaining the ignore, so we can find/fix later.